### PR TITLE
update views

### PR DESCRIPTION
**Pull request by wdhdev**
There is no reason for them to hover as they don't link to anything

### CODE REVIEW

**Review by Facinorous-420 — CHANGES_REQUESTED** [review]
Changes look good, and it makes sense to remove the hover in this area.

If we remove it from the admin page, we should also remove it from the user profile page stat cards. If you add this to this PR, I will happily add this into the v1.1.1 which will include the docker install.

Thanks for the contribution!

**Comment by wdhdev** [comment]
> Changes look good, and it makes sense to remove the hover in this area.
> 
> If we remove it from the admin page, we should also remove it from the user profile page stat cards. If you add this to this PR, I will happily add this into the v1.1.1 which will include the docker install.
> 
> Thanks for the contribution!

I forgot to remove it from the user profile. Will do this tomorrow.

File: views/pages/admin/tabs/users.ejs [hunk 1]
```diff
@@ -86,17 +86,13 @@
                                     <%= user.username %>
                                 </span>
                                 <div class="text-color-tertiary text-xs">
-                                    <%= user.role %>
+                                    <%= if(user.role === "admin"){ %> Administrator <% } else { %> User <% } %>
```
Comment (Facinorous-420) [inline views/pages/admin/tabs/users.ejs]: I personally think we should leave this the way it is, as it's a secondary text under the name and I think adding capitalization would make it look like it's own main text.

`String.toUpperCase()` is a wonderful method for doing this though, if we were to add it. I will think about this for the v2 react rewrite.

File: views/pages/user/main.ejs [hunk 2]
```diff
@@ -14,7 +12,7 @@
                     <%= user.username %>
                 </div>
                 <div class="text-xs text-color-secondary sm:text-base md:text-base lg:text-base">
-                    <% if(hasRole){ %> admin <% } else { %> user <% } %>
+                    <% if(hasRole){ %> Administrator <% } else { %> User <% } %>
```
Comment (Facinorous-420) [inline views/pages/user/main.ejs]: Same comment applies here, don't like the change of adding capitalization to these secondary text areas.
Please revert these 2 spots back and I will merge to v1.1.1 and create the new release! Thanks for the help!

**Review by Facinorous-420 — CHANGES_REQUESTED** [review]
Please remove the two spots you changed the user role texts to be capitalized, and everything looks perfect. 
Thanks.

**Comment by wdhdev** [comment]
Done. I also removed the showing [amount] of users in the admin users tab, since it just seems out of place and the average server probably doesn't have more than 2 or so users.

**Review by Facinorous-420 — APPROVED** [review]
Looks good thanks for the contribution

File: views/templates/public.ejs [hunk 3]
```diff
@@ -7,7 +7,7 @@
       		<% if(settingsDatabase.registrationEnabled) { %>
         		<% if(path == '/register'){ %> <%- include("../pages/public/register") %> <% } %> 
       		<% } %>
-      		<!-- /Page Content -->
+      		<!-- Page Content -->
```
Comment (Facinorous-420) [inline views/templates/public.ejs]: Please make this say End Page Content to represent this is the ending of this section

**Review by Facinorous-420 — CHANGES_REQUESTED** [review]

File: views/templates/public.ejs (hunk 3 above)
Comment (wdhdev) [reply views/templates/public.ejs]: @Facinorous-420 I reverted that change

**Review by Facinorous-420 — APPROVED** [review]
Changes look good

**Comment by wdhdev** [comment]
I'm all done, you can merge now.

**Pull request merged**